src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -165,7 +165,7 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 		}
 
 		const advancedTab = <DesignerTab>{
-			title: localize('tableDesigner.advancedTab', "Advanced"),
+			title: localize('tableDesigner.generalTab', "General"),
```

Comment:
Not going to update the var names?

Reply:
good catch, addressing this in https://github.com/microsoft/azuredatastudio/pull/17595